std/haxe/Http.hx

```diff
@@ -196,7 +198,66 @@ class Http {
 	**/
 	public function request( ?post : Bool ) : Void {
 		var me = this;
-	#if js
+	#if nodejs
+		me.responseData = null;
+		var url_regexp = ~/^(https?:\/\/)?([a-zA-Z\.0-9_-]+)(:[0-9]+)?(.*)$/;
```

Comment:
Would probably make it easier to maintain. I just copied the regex from the customRequest.